doc-content/enterprise-only/integration/sso-central-proc.adoc

```diff
@@ -10,7 +10,8 @@ This chapter describes how to authenticate {CENTRAL} through RH-SSO. It includes
 .Prerequisites
 * {CENTRAL} is installed in a {EAP} {EAP_VERSION} server, as described in {URL_INSTALLING_AND_CONFIGURING}#assembly-install-on-eap[_{INSTALLING_ON_EAP}_].
 * RH-SSO is installed as described in <<sso-realm-proc>>.
-* {CENTRAL} users have been added to RH-SSO as described in <<sso-user-add-proc>>.
+* You added {CENTRAL} users to RH-SSO as described in <<sso-user-add-proc>>.
+* Optional: In order to be able to manage RH-SSO users from {CENTRAL}, you added all realm-management client roles in RH-SSO to the {CENTRAL} administrator user.
```

Comment:
For minimalism, I would reword as follows:

Optional: In order to manage RH-SSO users from {CENTRAL}, you added all realm-management client roles in RH-SSO to the {CENTRAL} administrator user.